app/Transformers/CommentTransformer.php

```diff
@@ -42,6 +42,7 @@ public function transform(Comment $comment)
             'updated_at' => json_time($comment->updated_at),
 
             'deleted_at' => json_time($comment->deleted_at),
+            'deleted_by_id' => $comment->deleted_by_id,
```

Comment:
I haven't checked this PR but this field probably should only be visible for moderators

Reply:
Where should I do the check for moderators?

Reply:
check how `message` include works

Reply:
Done